tests/small/test_environ.py

```diff
@@ -77,33 +78,35 @@ def test_getSplunkPaths(default_yml, os_env, output):
 @pytest.mark.parametrize(("default_yml", "os_env", "output"),
             [
                 # Check null parameters
-                ({}, {}, {"label": None, "secret": None, "replication_factor": 1, "search_factor": 1}),
+                ({}, {}, {"pass4SymmKey": None, "label": None, "secret": None, "replication_factor": 1, "search_factor": 1}),
                 # Check default.yml parameters
-                ({"idxc": {}}, {}, {"label": None, "secret": None, "replication_factor": 1, "search_factor": 1}),
-                ({"idxc": {"label": None}}, {}, {"label": None, "secret": None, "replication_factor": 1, "search_factor": 1}),
-                ({"idxc": {"label": "1234"}}, {}, {"label": "1234", "secret": None, "replication_factor": 1, "search_factor": 1}),
-                ({"idxc": {"secret": None}}, {}, {"label": None, "secret": None, "replication_factor": 1, "search_factor": 1}),
-                ({"idxc": {"secret": "1234"}}, {}, {"label": None, "secret": "1234", "replication_factor": 1, "search_factor": 1}),
-                ({"idxc": {"secret": None}}, {}, {"label": None, "secret": None, "replication_factor": 1, "search_factor": 1}),
-                ({"idxc": {"secret": "1234"}}, {}, {"label": None, "secret": "1234", "replication_factor": 1, "search_factor": 1}),
+                ({"idxc": {}}, {}, {"pass4SymmKey": None, "label": None, "secret": None, "replication_factor": 1, "search_factor": 1}),
+                ({"idxc": {"label": None}}, {}, {"pass4SymmKey": None, "label": None, "secret": None, "replication_factor": 1, "search_factor": 1}),
+                ({"idxc": {"label": "1234"}}, {}, {"pass4SymmKey": None, "label": "1234", "secret": None, "replication_factor": 1, "search_factor": 1}),
+                ({"idxc": {"secret": None}}, {}, {"pass4SymmKey": None, "label": None, "secret": None, "replication_factor": 1, "search_factor": 1}),
+                ({"idxc": {"secret": "1234"}}, {}, {"pass4SymmKey": "1234", "label": None, "secret": "1234", "replication_factor": 1, "search_factor": 1}),
+                ({"idxc": {"secret": None}}, {}, {"pass4SymmKey": None, "label": None, "secret": None, "replication_factor": 1, "search_factor": 1}),
+                ({"idxc": {"secret": "1234"}}, {}, {"pass4SymmKey": "1234", "label": None, "secret": "1234", "replication_factor": 1, "search_factor": 1}),
```

Comment:
This is probably my mistake, but lines 86-87 are identical to lines 88-89. Maybe we can change 88/89 from "secret" to "pass4SymmKey"?
```
{"idxc": {"pass4SymmKey": "1234"}}
```

Reply:
Yep - the dangers of copy/paste. Will update